src/detection/cpu/cpu.c

```diff
@@ -24,6 +24,9 @@ const char* ffCPUAppleCodeToName(uint32_t code)
     // https://github.com/AsahiLinux/docs/wiki/Codenames
     switch (code)
     {
+        case 10: return "PowerPC 7400";
+        case 11: return "PowerPC 7450";
+        case 100: return "PowerPC 970";
```

Comment:
This function is only for Apple Silicon code name conversion. Please don't mess it up.

Reply:
I haven’t found any instances of its usage, perhaps missed it. Anyway, sure, I can make a separate one with PowerPC values.

Reply:
It's used for Asahi Linux, not for macOS

Reply:
Ah, got it. I will remove ppc from here and make its on function for it.